### PR TITLE
update README.md file

### PR DESCRIPTION
**Pull request by bhupathi-skyflow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -409,25 +502,30 @@ Skyflow Elements can be used to securely reveal data in an application without e
...
+        inputStyles = Skyflow.Styles(),        //optional, styles to be applied to the element
+        labelStyles = Skyflow.Styles(),  //optional, styles to be applied to the label of the reveal element
+        errorTextStyles = Skyflow.Styles(),  //optional styles that will be applied to the errorText of the reveal element
+        label = "cardNumber"            //optional, label for the element,
+        altText = "XXXX XXXX XXXX XXXX" //optional, string that is shown before reveal, will show token if altText is not provided
     )
 ```
-The `styles` parameter accepts a styles object as described in the [previous section](#step-2-create-a-collect-element) for collecting data but the only state available for a reveal element is the base state. For a list of acceptable redaction types, see the [section above](#Retrieving-data-from-the-vault).
+The `inputStyles` parameter accepts a styles object as described in the [previous section](#step-2-create-a-collect-element) for collecting data but the only state available for a reveal element is the base state. For a list of acceptable redaction types, see the [section above](#Retrieving-data-from-the-vault).
```
Comment (saketh-skyflow) [inline README.md:521]: can you add the variants available for input styles, error styles and label styles with sample code snippets

**Review by saketh-skyflow — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -10,45 +10,77 @@ Skyflow’s android SDK can be used to securely collect, tokenize, and display s
...
 # Installing skyflow-android
 ---
+
+## Step 1: Generate a Personal Access Token for GitHub
+- Inside you GitHub account:
+- Settings -> Developer Settings -> Personal Access Tokens -> Generate new token
+- Make sure you select the following scopes (“read:packages”) and Generate a token
+- After Generating make sure to copy your new personal access token. You cannot see it again! The only option is to generate a new key.
+
+## Step 2: Store your GitHub — Personal Access Token details
```
[the remark this answers is not in the document]
Comment (bhupathi-skyflow) [reply README.md:20]: for building the dependency locally, they need PAT locally, so they have to store it locally in properties file or as an system environment variable.

File: README.md [hunk 3]
```diff
@@ -10,45 +10,77 @@ Skyflow’s android SDK can be used to securely collect, tokenize, and display s
...
-      <artifactId>skyflow-android-sdk</artifactId>
-      <version>Tag</version>
-  </dependency>
-  ```
+- Add the package dependencies to the dependencies element of your project pom.xml file
+```xml
+<dependency>
+   <groupId>com.skyflowapi.android</groupId>
+   <artifactId>skyflow-android-sdk</artifactId>
+   <version>1.0.0</version>
```
[the remark this answers is not in the document]
Comment (bhupathi-skyflow) [reply README.md:81]: won't we update the readme file for every public release?, then we can change it to whatever version we are releasing.

File: README.md [hunk 4]
```diff
@@ -438,44 +536,53 @@ Elements used for revealing data are mounted to the screen the same way as Eleme
...
+        redaction = Skyflow.RedactionType.PLAIN_TEXT
+        inputStyles = inputStyles,
+        labelStyles = labelStyles,
+        errorTextStyles = errorTextStyles,
+        label = "cvv",
+        altText = "XXX"
 )
 
-val cvvElement = container.create(context:Context,input: cvvInput)
+val cvvElement = container.create(context = Context,input = cvvInput)
```
[the remark this answers is not in the document]
Comment (bhupathi-skyflow) [reply README.md:585]: This is the convention in android, previously we didn't notice this.

File: README.md (hunk 1 above)
Comment (bhupathi-skyflow) [reply README.md:521]: added in the create a collect element section, also mentioned here that only the base variant is applied.

File: README.md [hunk 5]
```diff
@@ -10,45 +10,77 @@ Skyflow’s android SDK can be used to securely collect, tokenize, and display s
...
+- Add properties gpr.usr=GITHUB_USER_NAME and gpr.key=PERSONAL_ACCESS_TOKEN
+- Replace GITHUB_USER_NAME with personal / organisation Github user NAME and PERSONAL_ACCESS_TOKEN with the token generated in #Step 1
+
+Alternatively you can also add the GPR_USER and GPR_API_KEY values to your environment variables on you local machine or build server to avoid creating a github properties file
+
+## Step 3: Adding the dependency to the project
+
 ### Using gradle
 
 - Add the JitPack repository to your root project build.gradle file
```
[the remark this answers is not in the document]
Comment (bhupathi-skyflow) [reply README.md]: okay, updated.

File: README.md [hunk 6]
```diff
@@ -10,45 +10,77 @@ Skyflow’s android SDK can be used to securely collect, tokenize, and display s
...
+- Make sure you select the following scopes (“read:packages”) and Generate a token
+- After Generating make sure to copy your new personal access token. You cannot see it again! The only option is to generate a new key.
+
+## Step 2: Store your GitHub — Personal Access Token details
+- Create a github.properties file within your root Android project
+- In case of a public repository make sure you add this file to .gitignore for keep the token private
+- Add properties gpr.usr=GITHUB_USER_NAME and gpr.key=PERSONAL_ACCESS_TOKEN
+- Replace GITHUB_USER_NAME with personal / organisation Github user NAME and PERSONAL_ACCESS_TOKEN with the token generated in #Step 1
+
+Alternatively you can also add the GPR_USER and GPR_API_KEY values to your environment variables on you local machine or build server to avoid creating a github properties file
```
[the remark this answers is not in the document]
Comment (bhupathi-skyflow) [reply README.md]: okay, updated.

**Pull request merged**